### PR TITLE
[skip-ci] update release notes

### PR DESCRIPTION
**Pull request by couet**
Update release notes

### CODE REVIEW

File: README/ReleaseNotes/v634/index.md [hunk 1]
```diff
@@ -158,6 +158,8 @@ This should have minimal impact on users, since one should usual use Minuit 2 vi
 
 * Initial error values can be used for initializating the Hessian matrix to be used in Minuit2 minimization algorithms by setting the `RooMinimizer::Config` option `setInitialCovariance` to `true`. These values correspond to the diagonal entries of the initial covariance matrix.
 
+* `THStack:GetMinimum()` was not correct in case of negative contents.
```
Comment (dpiparo) [inline README/ReleaseNotes/v634/index.md:161]: Why is this bullet under RooFit?

File: README/ReleaseNotes/v634/index.md [hunk 2]
```diff
@@ -223,7 +225,18 @@ TCanvas::SaveAll({c1, c2, c3, c4}, "file.pdf");
 ```
 
 ## 2D Graphics Libraries
+* In `TGraphErrors` `TGraphAsymmErrors` and `TGraphBentErrors`, the error bars were drawn
```
Comment (dpiparo) [inline README/ReleaseNotes/v634/index.md:228]: Is there some line break kicking in? Do we maybe want to have also these bullets on one single lines like all others?

File: README/ReleaseNotes/v634/index.md [hunk 3]
```diff
@@ -223,7 +225,18 @@ TCanvas::SaveAll({c1, c2, c3, c4}, "file.pdf");
...
+  inside the marker when the marker was bigger than the error bars. This produced a weird
+  plot. This is now fixed.
+
+ When error-bars exceeded the y range limits the end of error bars were nevertheless displayed
+  on the x-bottom and top axis. So it looked like the total error bar while it was indeed not.
+
+* New color schemes have been implemented as described [here](https://root.cern.ch/blog/new-color-schemes/).
```
Comment (dpiparo) [inline README/ReleaseNotes/v634/index.md:235]: Maybe we could directly use the word "incusive" and cite the presentation at ICHEP?
What about texgyre fonts?

File: README/ReleaseNotes/v634/index.md [hunk 4]
```diff
@@ -326,6 +339,8 @@ JSROOT is now used as default display in `jupyter`.
 
 ## Tutorials
 
+* New tutorials [accessiblecolorschemes.C](https://root.cern/doc/master/accessiblecolorschemes_8C.html)
```
Comment (dpiparo) [inline README/ReleaseNotes/v634/index.md:342]: See above for the multiline bullets.

**Review by dpiparo — CHANGES_REQUESTED** [review]

File: README/ReleaseNotes/v634/index.md (hunk 1 above)
Comment (couet) [reply README/ReleaseNotes/v634/index.md:161]: Mistake. I meant to put in in the Histograms section

File: README/ReleaseNotes/v634/index.md (hunk 3 above)
Comment (couet) [reply README/ReleaseNotes/v634/index.md:235]: Tex Gyre was already mentionned in v632  release notes

**Review by dpiparo — APPROVED** [review]
Ready to merge and backport to 6.34. Thanks

**Pull request merged**